Meet-The-Admins Section

mmcbride2929 opened the pull request:
# TL/DR
This PR updates the admin section by adding new admin photos + their titles

# Overview of Change

`Members.jsx`
- Add new members + their titles
- Add `<h5>` for newly added member titles

`index.css`
- Commented out padding .75em from `.member` class (temporary based on opinion)
- Add styling for admins name and titles (h4 & h5's)
- change `member` className to `admin/member-avatar`
- edit array so we can seperate first and last name
- add id to admin so we always have unique keys


With Padding:
![admins-padding](https://github.com/FCCColumbus/cbus-web/assets/77947488/bd870138-3657-4d5c-b39d-2594fae17d1d)


Without Padding:
![admins-no-padding](https://github.com/FCCColumbus/cbus-web/assets/77947488/cdd913e3-acb0-4ae3-932f-7c74340204ba)

jduffey commented:
This PR is asking to merge into the upstream `main` instead of `dev`. Unless there's a way to change that after the PR has been opened, you'll need to close this PR and open a new one. At a quick glance and `npm test && npm start`, it looks great! I'll do a formal review of the actual code changes once I have more time and the PR is pointing toward the correct upstream target (`dev`).

readysetagile commented:
yeah I had to change the default branch back to main because of #57 .  Another side-effect is that we will have to be aware of the base branch where our PRs will go.

It's also easy to change the base branch.  You can just edit the PR title and you will get the option

readysetagile commented:
+1 on the "no padding".  it looks a lot better.  

Also, I think that a "forced" new line (a `<br>` element) in-between everyone's first and last name looks better IMO.

![image](https://github.com/FCCColumbus/cbus-web/assets/41979303/df472868-91fe-477f-8ca4-74147ade1a2d)

jduffey commented on src/index.css:
```diff
@@ -201,7 +201,7 @@ th {
   height: 140px;
   width: 140px;
-  padding: .75em;
+  /* padding: .75em; */
```
I'm assuming this is still WIP but I might refactor this and the related avatar styles to more clearly separate the styles per class. I also like putting the word "avatar" on the class name to more immediately make clear what precisely the style is referring to:

```
.avatar {
  border: 1px solid #707070;
  background-color: #f5b742;
}

.admin-avatar {
  height: 140px;
  width: 140px;
}

.member-avatar {
  height: 100px;
  width: 100px;
  border-radius: 50%;
}
```

You'll of course want to apply both of these styles to the relevant avatar rows, .e.g:

`<img className='avatar admin-avatar' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />`

jduffey replied:
At time of this comment it looks like this:

```
.admin,
.member {
  height: 140px;
  width: 140px;
  /* padding: .75em; */
  border: 1px solid #707070;
  background-color: #f5b742;
}

.member {
  height: 100px;
  width: 100px;
  border-radius: 50%;
}
```

The issue here is that the initial assignment of the height/width to `.member` is overwritten in the next style block. In my previous comment I've refactored these two blocks into three styles: one for the styles that are common among both admins and members, and then one each for the styles that are unique to each.

mmcbride2929 commented:
I have updated the css & made a new entry for the admin's first and last name values. I have also added the name above the photo because I thought it looked a little better. Let me know what you think, if it doesn't work i'll be happy to switch it back.

Before
![admins-with-name-above](https://github.com/FCCColumbus/cbus-web/assets/77947488/fd1599bb-6a0b-4628-8557-197c1b160302)

After
![admins-with-title](https://github.com/FCCColumbus/cbus-web/assets/77947488/bc526384-bcaf-4071-be2b-fa33c6673517)

jduffey commented on src/components/Members.jsx:
```diff
@@ -18,19 +22,20 @@ function Members() {
-            <div key={member.name}>
-              <img className='admin' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
-              <h4>{member.name}</h4>
+            <div key={member.id}>
```
Good job catching this and adding the key. I'm guessing you maybe noticed a warning in the browser console? I'm usually good at remembering to test with my console open but have been forgetting about it recently.. 😑

mmcbride2929 replied:
Happens to the best of us!

jduffey commented:
> I'm good with the names on top.
> 
> @jduffey are you going to review or are you waiting for me?
> 
> Also, FYI, this section is not "responsive", that is, the section does not scale with the changing dimensions. It is not a requirement for the site or this issue yet, so **no changes needed**. We will have to revisit this at a later time to make decisions on the responsiveness..

@readysetagile & @mmcbride2929 I have two pending comments. Overall things look great, I'm happy with the new design.

jduffey reviewed (approved):
Approved. Looks great and thanks for addressing the second round of comments.
